rust/psibase/src/services/invite.rs

```diff
@@ -18,11 +17,8 @@ pub struct InviteRecord {
     state: u8,
 }
 
-#[derive(
-    Debug, Copy, Clone, Pack, Unpack, Reflect, Serialize, Deserialize, SimpleObject, InputObject,
-)]
+#[derive(Debug, Copy, Clone, Pack, Unpack, Serialize, Deserialize, SimpleObject, InputObject)]
```

Comment:
Was `ToSchema` intended here?

Reply:
It wouldn't hurt to add `ToSchema`, but it isn't actually needed yet, since `NewAccountRecord` isn't used in any action or event signatures.